handler/release_handler.go

```diff
@@ -104,7 +100,13 @@ func updateReleaseNotes(client *github.Client, owner, repo, latestTag string) er
 		output = output + fmt.Sprintf("%s %s%s\n", id, title, author)
 	}
 
-	output = fmt.Sprintf("%s\nGenerated by [Derek](https://github.com/alexellis/derek/)\n", output)
+	var releaseDiff string
+	if workingReleases.PreviousTag != "" {
+		releaseDiff = fmt.Sprintf("Changes: https://github.com/%s/%s/compare/%s...%s", owner, repo, workingReleases.PreviousTag, workingReleases.CurrentTag)
+	} else {
+		releaseDiff = fmt.Sprintf("Changes: https://github.com/%s/%s/compare/%s...%s", owner, repo, *includedCommits[len(includedCommits)-1].SHA, workingReleases.CurrentTag)
```

Comment:
What does this print?

The last but one commit included? I'm trying to picture it but struggling.

Reply:
here is an example: https://github.com/cpanato/temporary-playground/releases/tag/0.0.11

It prints the sha of the first commit and the tag

Reply:
`includedCommits[len(includedCommits)-1]` <- that looks like the last included commit in the slice? Is that due to sorting/ordering?

This could be documented better by using a comment or using a named variable.